spec/chewy/config_spec.rb

```diff
@@ -94,4 +95,15 @@ def self.root
       end
     end
   end
+
+  describe '.console_strategy' do
+    context 'sets .console_strategy' do
+      let(:strategy) { :atomic }
```

Comment:
Should we use the `after` hook to prevent the leak of the state?

Reply:
@rabotyaga Thank you for noticing this. 
I added the `after` hook.